src/renderer/domains/collectives/salary/service.ts

```diff
@@ -139,16 +139,12 @@ function isSalaryInductTransaction(transaction: Transaction): transaction is Sal
   return transaction.type === TransactionType.COLLECTIVE_SALARY_INDUCT;
 }
 
-function isClaimantActiveInCurrentCycle(claimStatus: ClaimStatus, period: SalaryCyclePeriod) {
-  return claimStatus.lastActive === period.cycleIndex;
+function isClaimantRequestedSalary(claimStatus: ClaimStatus) {
+  return claimStatus && claimStatus.type === 'registered';
 }
```

Comment:
if it is nullable then we can maybe reflect it in the types here or handle the emptiness outside of this function